### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by mattcollier**
@dlongley need to discuss fixmes here when you're available.

### CODE REVIEW

File: test/mocha/12-insert.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    try {
+      record = await brEdvStorage.insert({
+        actor,
+        edvId: 'urn:uuid:something-else',
+        doc: mockData.doc1
+      });
+    } catch(e) {
+      err = e;
+    }
+    console.log('RRRRRRRRr', record);
```
Comment (aljones15) [inline test/mocha/12-insert.js]: You did leave a console.log here.

File: test/mocha/20-http.js [hunk 2]
```diff
@@ -93,10 +91,26 @@ describe('bedrock-edv-storage HTTP API', () => {
...
+        urls.edvs, mockConfig,
+        {headers: {
+          'x-test-account': 'alpha@example.com'
+        }});
+      documentEndpoint = `${response.data.id}/documents`;
+    });
     it('should insert a document', async () => {
       const response = await axios.post(
-        urls.documents, mockData.doc1,
+        documentEndpoint, mockData.doc1,
```
Comment (aljones15) [inline test/mocha/20-http.js]: this is definitely an improvements in terms of naming.

**Comment by aljones15** [comment]
@mattcollier Hey this runs just fine for me. I am seeing 29 passing and 28 pending, but it definitely does run. Good work.

**Comment by mattcollier** [comment]
OK, that's going to have to do for this pass.

**Pull request merged**